pkg/bgs

```diff
@@ -0,0 +1,21 @@
+[mirrors]
+https://github.com/Gottox/bgs/archive/master.tar.gz
+
+[vars]
+filesize=4991
+sha512=30c806c097a6fdf2afd9a17a2cc1c2a27cce65df17a33921952ac68946b2a0efbcabcff00fab67013989478844e00c17899b4036bbd69dd7a110789ee4b2622c
+pkgver=1
+tardir="bgs-master"
+desc="simple background setter based on imlib2"
+
+[deps]
+imlib2
+
+[build]
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
+
+
+make V=1 -j$MAKE_THREADS
+make PREFIX="$butch_install_dir" MANPREFIX="$butch_install_dir" install
```

Comment:
are you sure those installation targets are correct ? also even if they are, you forgot $butch_prefix (which a user may have chosen to set to /usr, for example, instead of using the default)